fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -1490,6 +1502,14 @@ protected void process() {
             checkCompatibility();
         }
 
+        if (translator.isDerby() && !"APP".equals(schema.getSchemaName())) {
+            if (!schema.isDefaultSchemaName()) {
```

Comment:
1505 the last conditional ... is that the same as the 1506?

Reply:
no.  1505 checks whether schema.getSchemaName = APP.
if so, then we don't need to do anything.
if not, then we want to set it to APP (and log a warning iff the value was explicitly passed)